installer/hyper_install.c

```diff
@@ -184,7 +184,7 @@ int main(int argc, char **argv)
     img = safe_open(argv[1], "r+b");
 
     read_mbr_partition_list(img, mbr_partitions);
-    is_iso = is_iso_disk(img);
+    is_iso = is_iso || is_iso_disk(img);
```

Comment:
`is_iso |= is_iso_disk(img)`

Reply:
Change from my own testing I forgot to remove -- fixed